lib/slice.ark

```diff
@@ -0,0 +1,16 @@
+{
+  (let slice (fun (string escapeCharacter){
+    #made by Natendrtfm <3
+        (mut i 0)
+        (mut word "")
+        (mut letter "")
+        (mut sliccedList [])
```

Comment:
rename to `output` or `wordList`, since it's not a slice

Reply:
Done, renamed it to output